build.sh

```diff
@@ -180,6 +186,10 @@ if buildAll || hasArg libucxx; then
     compile_end=$(date +%s)
     compile_total=$(( compile_end - compile_start ))
 
+    if [[ ${BUILD_COMPILE_COMMANDS} == "ON" ]]; then
+      cp compile_commands.json ..
```

Comment:
If the build directory is `cpp/build` then clangd automatically finds `compile_commands.json`, so there's not necessarily any need to do this copy.

Reply:
Hmm, at least for me clangd via [neovim+LSP](https://github.com/neovim/nvim-lspconfig) hasn't been finding it in `cpp/build`, it's possible I missed some config but couldn't get it to work oterhwise so far.

Reply:
Ah, no worries.